tests/script-based-pre/playback_already_existing/original.rs

```diff
@@ -0,0 +1,35 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+//! Check that Kani correctly adds tests to the cover checks reachable in a harness.
+extern crate kani;
+
+#[cfg(kani)]
+mod verify {
+    use kani::cover;
+    use std::convert::TryFrom;
+    use std::num::NonZeroU8;
+
+    #[kani::proof]
+    fn try_nz_u8() {
+        let val: u8 = kani::any();
+        let result = NonZeroU8::try_from(val);
+        match result {
+            Ok(nz_val) => {
+                cover!(true, "Ok");
+                assert_eq!(nz_val.get(), val);
+            }
+            Err(_) => {
+                cover!(true, "Not ok");
+                assert_eq!(val, 0);
+            }
+        }
+    }
+    #[test]
+    fn kani_concrete_playback_try_nz_u8_17663051139329126359() {
```

Comment:
I would actually suggest that you run concrete playback twice in a row, instead of relying on the name staying the same.

Reply:
Nice idea. Will implement in the script.

Reply:
5979461e053a02b480dd3d5a81cc6af5ad400611